recipe/meta.yaml

```diff
@@ -22,12 +22,13 @@ requirements:
     - {{ compiler("c") }}
     - {{ stdlib("c") }}
     - {{ compiler("cxx") }}
-    - {{ cdt("systemd-devel") }}
```

Comment:
I'm not confident I know/understand why `systemd-devel` was needed before and is not anymore? Was that unnecessary all the time or was there anything that happened lately that now removes that requirement?

Reply:
There were some issues while making these changes where it wasn't detecting `libsystemd` headers. Part of this was addressed by updating the CMake build above

However the other issue is `libsystemd` and `libudev` were in `requirements/build` instead of `requirements/host`. As these are libraries we are linking to, they should be in `requirements/host`. So have made that change below: https://github.com/conda-forge/rdma-core-feedstock/pull/25#discussion_r1823117662

Reply:
Thanks for addressing that.